packages/manager/fragments/navbar/src/language/button.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+// eslint-disable-next-line react/display-name
```

Comment:
Looks like we are not referring to the right ESLint configuration here because I noticed some `eslint-disable*` comments.

Maybe we could add `eslint-config-airbnb` as a devDependency.

```sh
$ yarn workspace @ovh-ux/manager-fragment-navbar add -D eslint-config-airbnb eslint-plugin-jsx-a11y eslint-plugin-react-hooks
```

see: https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/rules/react.js#L51